Zend/zend_builtin_functions.c

```diff
@@ -1009,7 +1009,7 @@ ZEND_FUNCTION(get_class)
 {
 	zval *obj = NULL;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o!", &obj) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|o", &obj) == FAILURE) {
```

Comment:
@antanas-arvasevicius get_called_class is aware of the 'late static binding' aka the real class, not the current scope class, so they are not equivalent. https://3v4l.org/ISkEZ